packages/terra-form-select/src/_Menu.jsx

```diff
@@ -0,0 +1,291 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { polyfill } from 'react-lifecycles-compat';
+import 'terra-base/lib/baseStyles';
+import { KeyCodes, Variants } from './_constants';
+import AddOption from './_AddOption';
+import NoResults from './_NoResults';
+import Util from './_MenuUtil';
+import styles from './_Menu.module.scss';
+
+const cx = classNames.bind(styles);
+
+/* Disabling this rule because props used in getDerivedStateFromProps return false positives. */
+/* eslint-disable react/no-unused-prop-types */
+const propTypes = {
+  /**
+   * The content of the menu.
+   */
+  children: PropTypes.node,
+  /**
+   * Content to display when no results are found.
+   */
+  noResultContent: PropTypes.node,
```

Comment:
There was a comment on the tech design about this being plural or singular. Does anyone have an opinion on this? 

Potential Options include, but not limited to. ( With or without the Content suffix )
`noResultContent`
`noResultsContent`
`notFoundContent`